<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDiagnostic</class>
 <widget class="QWidget" name="FormDiagnostic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Введите симптомы</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Clump thickness</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Uniformity of cell size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Uniformity of cell shape</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Marginal adhesion</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Single epithelial cell size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Bare nuclei</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Bland chromatin</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Normal nucleoli</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Mitoses</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Получить диагноз</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="edit_adhesion">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="edit_barenuclei">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="edit_cellshape">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="edit_cellsize">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="edit_chromatin">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="edit_clump">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="edit_epithelialcellsize">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="edit_mithoses">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="edit_nucleoli">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
